debian/daos-server.postinst

```diff
@@ -1,9 +1,9 @@
 #!/bin/sh
 set -e
 
-chown daos_server:daos_admins /etc/daos/certs /etc/daos/certs/clients
+chown daos_server:daos_server /etc/daos/certs /etc/daos/certs/clients
```

Comment:
Hmm. This isn't a result of your PR, but I'm not sure these permissions are correct. @dpquigl: Shouldn't these certificate directories be owned/writable by root and only readable by the daos_server group?

Reply:
You're not setting any permissions here ... just user/group ownership.

That said, I agree that permissions should be set explicitly. Better done in another PR, as there are more places that could gain from explicit permission settings...

Reply:
Yes, sorry, I meant ownership in this specific case. And I agree that we can clean this up in a separate PR; I just happened to notice it while reviewing this one. The debian packaging was added relatively recently.

Reply:
I've been talking with Phil Henderson about the permissions and ownership on these files and directories. We have an issue where you want to install the client and server packages on the same host. Based on my conversation with phil I think we need to make /etc/daos/certs more permissive so people can get into it. Have each of the certs owned by the respective users (server by daos_server, agent by daos_agent and keep caRoot as owned by root but readable by everyone) and make the clients directory owned by daos_server since its the only one that is going to use it. The other option is to have agent and server directories under /etc/daos/certs or have agent_certs and server_certs so it can be locked down further. There isn't really a good answer for this since they are technically two different security domains and really shouldn't be sharing the space but its inconvenient to have them not share it.

Reply:
Good point about the certs. As far as I understand it, utils/certs/gen_certificates.sh today sets permissions of the files that it creates, but it does not their set user or group. So the files will be owned by the user who calls the script (root?). To make the intended behavior explicit, shouldn't that script set the user and group for each of the files it creates?
And debian/daos-server.postinst is doing the opposite: it sets user and group of the cert directories, but fails to sets their permissions.

All of this ambiguity exists independently from my patch in this PR, which is really only renaming the daos_admins group to daos_server as far as the certs are concerned... Happy to add patches to disambiguate the user/group/permission in the certs area under the same PR, if we agree what they should be.

Reply:
The problem is you could generate certs on a platform that doesn't have any of the DAOS uses and groups. This is why they should be set explicitly in the RPM spec and DEB scripts since those will be the ones that create the daos_server, daos_admins, and daos_agent users/groups.